shopinvader_api_cart/schemas/amount.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from extendable_pydantic import ExtendableModelMeta
+from pydantic import BaseModel
+
+from odoo import _
+from odoo.exceptions import UserError
+from odoo.tools.float_utils import float_round
+
+from odoo.addons.pydantic import utils
+
+
+class SaleAmount(BaseModel, metaclass=ExtendableModelMeta):
+    tax: float
+    untaxed: float
+    total: float
+    discount_total: float
+    total_without_discount: float
+
+    @classmethod
+    def from_orm(
+        cls, odoo_rec
+    ):  # TODO find another name. But can be applied on SO or SOL
```

Comment:
How to call a `from_orm` method that can be applied on 2 models?

Reply:
Maybe use 2 different functions?

Reply:
Done. Now we have `from_sale_order` and `from_sale_order_line`